shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -463,7 +463,25 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
 void FlutterPlatformViewsController::CompositeWithParams(int view_id,
                                                          const EmbeddedViewParams& params) {
   CGRect frame = CGRectMake(0, 0, params.sizePoints().width(), params.sizePoints().height());
-  UIView* touchInterceptor = touch_interceptors_[view_id].get();
+  FlutterTouchInterceptingView* touchInterceptor = touch_interceptors_[view_id].get();
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
+  FML_CHECK(CGPointEqualToPoint([touchInterceptor embeddedView].frame.origin, CGPointZero));
```

Comment:
Please make this a dcheck.

This will cause the application to crash in debug mode for end developers.

Reply:
(as opposed to a crash/test failure for unopt builds by engine developers)